src/rootCompletion/rootName.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as vscode from 'vscode'
+import { getSimpleCompletionItem, getPossibleItems } from './utils'
+
+export function checkForRootNameItems(
+  document: vscode.TextDocument,
+  position: vscode.Position
+): vscode.CompletionItem[] {
+  if (
+    !document.fileName.includes('launch.json') ||
+    !document.lineAt(position).text.includes('"rootName"')
+  )
+    return []
+
+  const possibleRootNames: string[] = getPossibleItems(
+    document,
+    position,
+    'xs:element',
```

Comment:
Looking for elements with the xs prefix is potential fragile. We sometimes see "xsd" as an alternative prefix, and more and more schemas are written so the "xs:" prefix isn't needed at all (e.g. `<element name="foo">...`. 

I would suggest either passing in a namespace instead of a prefix and then look for elements that have a prefix that maps to that namespace, or maybe more simply just ignore namespaces/prefixes and assume the schema document uses correct namespaces/prefixes. It looks like the way fast-xml-parser it makes dealing with namespaces/prefixes diffciult, I would suggest doing the latter and setting the fast-xml-parser option `removeNSPrefix` to true and not worry about them.